Story/7DeadlyDragons/Extra/HatchTheEgg.cs

```diff
@@ -51,16 +51,7 @@ public void Hatch()
         Story.MapItemQuest(6907, "void", 6453);
 
         //6908 | Void Energy Needed
-        if (!Story.QuestProgression(6908))
-        {
-            Core.EnsureAccept(6908);
-            while (!Bot.ShouldExit && !Core.CheckInventory(48632, 8))
-            {
-                Core.Join("void", "r11", "Left");
-                Bot.Kill.Monster("Void Elemental");
-            }
-            Core.EnsureComplete(6908);
-        }
+        Story.KillQuest(6908, "void", "Void Elemental");
```

Comment:
why changing this ? this is prolly was put bcuz of there is item name with the same name as the temp item